### PR TITLE
Fix customize notification example

### PR DESCRIPTION
**Pull request by jschaedl**

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -500,10 +500,10 @@ very high and the recipient has a phonenumber::
                 && $recipient instanceof AdminRecipient
                 && null !== $recipient->getPhone()
             ) {
-                return ['sms', 'email'];
+                return ['sms'];
             }
 
-            return 'email';
+            return ['email'];
```
Comment (jschaedl) [inline notifier.rst]: The `Notification::getChannels(...)` methods needs to return an array.

File: notifier.rst [hunk 2]
```diff
@@ -500,10 +500,10 @@ very high and the recipient has a phonenumber::
                 && $recipient instanceof AdminRecipient
                 && null !== $recipient->getPhone()
             ) {
-                return ['sms', 'email'];
+                return ['sms'];
```
Comment (jschaedl) [inline notifier.rst]: This example is about returning the sms channel only in case the price is high and the recipient has a phone number.

**Comment by OskarStark** [comment]
Can you change this line too:
```diiff
- very high and the recipient has a phonenumber::
+ very high and the recipient has a phone number::
```

**Comment by jschaedl** [comment]
@OskarStark fixed

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Jan.

**Pull request merged**